formatTest/unit_tests/expected_output/syntax.re

```diff
@@ -1284,10 +1284,10 @@ Thing.map(
   foo,
   bar,
   baz,
-  [@attr]
-  ((abc, z) => MyModuleBlah.toList(argument)),
-  [@attr]
-  ((abc, z) => MyModuleBlah.toList(argument)),
+  [@attr] (abc, z) =>
```

Comment:
This formatting is a little weird. We might consider improving it next time we're in this part of the formatter.

Reply:
How would you like to see this?